spec/lib/carma/models/submission/metadata_spec.rb

```diff
@@ -16,16 +98,70 @@
     end
 
     it 'sets request_payload_keys' do
-      expect(described_class.request_payload_keys).to eq([:claim_id])
+      expect(described_class.request_payload_keys).to eq(
+        %i[
+          claim_id
+          veteran
+          primary_caregiver
+          secondary_caregiver_one
+          secondary_caregiver_two
+        ]
+      )
     end
   end
 
   describe '#to_request_payload' do
     it 'can receive :to_request_payload' do
-      metadata = described_class.new claim_id: 123
-      expect(metadata.to_request_payload).to eq(
+      example_1 = described_class.new claim_id: 123
```

Comment:
If you have two examples, I think these should be two separate tests.
`it 'can receive :to_request_payload'`(example_2) and maybe `it 'can receive :to_request_payload with only claimId'`(example_1) or something like that?